build.sbt

```diff
@@ -15,16 +14,16 @@ resolvers += Resolver.bintrayRepo("hmrc", "releases")
 
 lazy val wartremoverSettings =
   Seq(
-    wartremoverErrors in (Compile, compile) ++= Warts.allBut(
+    (Compile / compile / wartremoverErrors) ++= Warts.allBut(
       Wart.DefaultArguments,
       Wart.ImplicitConversion,
       Wart.ImplicitParameter,
       Wart.Nothing,
       Wart.Overloading,
       Wart.ToString
     ),
-    wartremoverExcluded in (Compile, compile) ++=
-      routes.in(Compile).value ++
+    wartremover.WartRemover.autoImport.wartremoverExcluded ++=
```

Comment:
Do we need to do like this?
`(Compile / compile / wartremoverExcluded)`

Reply:
Yes thats the standard now.

Reply:
Another example Test / compile for test compilation exclusions

Reply:
Oops, there is one still not done at Line 29: Done it now.